graph/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2018"
 
 [dependencies]
 async-trait = "0.1.25"
-bigdecimal = { version = "0.0.14", features = ["serde"] }
+bigdecimal = { version = "0.1.0", features = ["serde"] }
```

Comment:
This looks like a fairly significant version bump. Is it necessary? I'm asking because there is some risk that they changed things in incompatible ways that would warrant a closer look at the the changes before we update.

Reply:
Yes, it is necessary. I went through the actual changes in this repository and verified that there are no breaking changes (If I remember there was just a minor new API added).

This resolves an issue with the build where our interop failed to compile since diesel also updated their dependency on bigdecimal.

Reply:
Ah we also updated diesel. I was suprised to see this bump not cause incompatibility.